tests/mocks/task.py

```diff
@@ -6,19 +6,16 @@
 
 
 class MockTask:
-    @staticmethod
-    def drive_to(
-        position: Position = Position(x=1, y=1, z=1, frame=Frame.Robot),
-        orientation: Orientation = Orientation(
-            x=0, y=0, z=0.7071068, w=0.7071068, frame=Frame.Robot
-        ),
-    ) -> DriveToPose:
-        return DriveToPose(
-            pose=Pose(position=position, orientation=orientation, frame=Frame.Robot)
+    drive_to = DriveToPose(
+        pose=Pose(
+            position=Position(x=1, y=1, z=1, frame=Frame.Robot),
+            orientation=Orientation(
+                x=0, y=0, z=0.7071068, w=0.7071068, frame=Frame.Robot
+            ),
+            frame=Frame.Robot,
```

Comment:
Is it possible to use the MockPose you made in mock/pose.py?

Reply:
Agreed, using the `MockPose.default_pose` as you suggest.